### DISCUSSION TITLE
How to close the hp window

### DISCUSSION THREAD

**Discussion by az-z** [airblade/vim-gitgutter#892] [General]
Andy,
 right now i use : <control> K to navigate to the hunk preview window, followed by <ESC>:q to close the window.

I wonder if there is shortcut to close the preview window without moving onto it?
Or, even better, is there a way to make the window close after the cursor moved away from the changed line?

Thank you
AZ

**Comment by airblade** [comment]
See `:help :GitGutterPreviewHunk` for information about all the different ways to closing non-floating and floating preview windows.

For example `:pc` closes the preview window from anywhere, as does `CTRL-W_z` or `CTRL-W_CTRL-Z`.

**Comment by airblade** [comment]
You could also add something like this to your vimrc:

```vim
au CursorMoved * if gitgutter#hunk#in_hunk(line(".")) | GitGutterPreviewHunk | else | pclose | endif
```
